includes/Api/ApiStoreSubtitle.php

```diff
@@ -37,8 +37,11 @@ public function register_store_subtitle_route() {
 			'wubtitle/v1',
 			'/store-subtitle',
 			array(
-				'methods'  => 'POST',
-				'callback' => array( $this, 'auth_and_get_subtitle' ),
+				'methods'             => 'POST',
+				'callback'            => array( $this, 'auth_and_get_subtitle' ),
+				'permission_callback' => function() {
+					return current_user_can( 'manage_options' );
+				},
```

Comment:
Is this endpoint get called when a new job is started? the permission used in permission_callback is for administrator only. Should this operation be called also for other roles? (such as authors and editors)